tests/in/padding.param.ron

```diff
@@ -18,5 +18,6 @@
 		inline_samplers: [],
 		spirv_cross_compatibility: false,
 		fake_missing_bindings: false,
+		zero_initialize_workgroup_memory: true,
```

Comment:
If you prefer, rather than editing all these `.ron` files, you could use the [`#serde(default = "path")`](https://serde.rs/field-attrs.html#default--path) attribute. I think the value has to be a function name - maybe add it to `back/mod.rs`?

Reply:
We don't seem to do it for any of the other members, not sure why but wanted to keep things consistent.
I think if we want to introduce defaults, we should do so for all members and override the defaults in the ron files.

Reply:
Maybe best left to a follow-up PR.